TodoApi.Web/Properties/launchSettings.json

```diff
@@ -3,7 +3,7 @@
     "windowsAuthentication": false,
     "anonymousAuthentication": true,
     "iisExpress": {
-      "applicationUrl": "http://localhost:2173/",
+      "applicationUrl": "http://localhost:5000/",
```

Comment:
I prefer odd numbers. Do you think we could keep this as an odd port number?

Reply:
No, your face is odd. Let's keep this even.

Reply:
No, 5000 works for me.

Reply:
No, that's silly